src/r2mm/downloading/BetterThunderstoreDownloader.ts

```diff
@@ -185,7 +189,7 @@ export default class BetterThunderstoreDownloader extends ThunderstoreDownloader
 
             try {
                 const response = await this._downloadCombo(combo, singleModProgressCallback);
-                await this._saveDownloadResponse(response, combo, singleModProgressCallback);
+                await this._saveDownloadResponse(response, combo, singleModProgressCallback); //TODO: track installation separately
```

Comment:
Not sure what you mean by this comment? `downloadImportedMods` already has this covered.